Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -234,33 +236,41 @@ HTMLLinkElement::LinkProcessingOptions HTMLLinkElement::create_link_options()
 JS::GCPtr<Fetch::Infrastructure::Request> HTMLLinkElement::create_link_request(HTMLLinkElement::LinkProcessingOptions const& options)
 {
     // 1. Assert: options's href is not the empty string.
+    VERIFY(options.href != "");
 
-    // FIXME: 2. If options's destination is not a destination, then return null.
+    // 2. If options's destination is null, then return null.
+    if (!options.destination.has_value())
+        return nullptr;
```

Comment:
I believe that this is the change that is causing the tests to fail. This should either be removed for now, or the correct `options.destination` value should be set `HTMLLinkElement::create_link_options()`.

Reply:
I think you're probably right, I'll leave this as a FIXME and handle destinations in another pr.